felix/cmd/calico-bpf/commands/policy_debug.go

```diff
@@ -138,19 +151,34 @@ func dumpPolicyInfo(cmd *cobra.Command, iface string, h hook.Hook, m counters.Po
 	cmd.Printf("Hook: %s\n", policyDbg.Hook)
 	cmd.Printf("Error: %s\n", policyDbg.Error)
 	cmd.Println("Policy Info:")
+
+	var rule_id string
 	for _, insn := range policyDbg.PolicyInfo {
 		for _, comment := range insn.Comments {
+			if strings.Contains(comment, "rule_id") {
+				rule_id = getRuleIdentfier(comment)
+			}
 			if strings.Contains(comment, "Rule MatchID") {
 				matchId := getRuleMatchID(comment)
-				cmd.Printf("// count = %d\n", m[matchId])
+				if verboseFlagSet {
+					cmd.Printf("// count = %d\n", m[matchId])
+				} else {
+					cmd.Printf("RuleID['%s'] => count = %d\n", rule_id, m[matchId])
```

Comment:
The rule id is a very opaque internal identificator that does not convey much information. In iptables the rule IDs are somewhat readable, but unfortunately not in eBPF. This is how a sample verbose output looks like:
```
// Start of policy default.self-snat

// Start of rule action:"allow" src_net:"172.18.0.6/32" rule_id:"5vusbnpUZ-y8FtmR" 

// count = 2

// If source not in {172.18.0.6/32}, skip to next rule

rule_0_no_match:

  6191000000000000 LoadReg32 dst=R1 src=R9 off=0 imm=0x00000000/0  R1 = *(u32 *)(R9 + 0) /* state->ip_src */

  b4020000ffffffff MovImm32 dst=R2 src=R0 off=0 imm=0xffffffff/-1                                  

  5c12000000000000 And32 dst=R2 src=R1 off=0 imm=0x00000000/0                                      

  16020100ac120006 JumpEqImm32 dst=R2 src=R0 off=1 imm=0x060012ac/100668076  goto rule_1_part_0

  05000c0000000000 JumpA dst=R0 src=R0 off=12 imm=0x00000000/0               goto rule_1_no_match

rule_1_part_0:
```

I think a user would like to see the whole `Start of rule action:"allow" src_net:"172.18.0.6/32" rule_id:"5vusbnpUZ-y8FtmR" ` perhaps without the `relu_id` but with the `policy default.self-snat` so he/she can match it with the policy in a yaml.

Reply:
yeah, that looks good, thanks

Reply:
Awesome thanks - will implement accordingly